db/migrate/20200622142118_add_creator_id_to_event.rb

```diff
@@ -0,0 +1,5 @@
+class AddCreatorIdToEvent < ActiveRecord::Migration[6.0]
+  def change
+    add_column :events, :creator_id, :integer
+  end
```

Comment:
**[REQUIRED]**
Let's add an index column for out event migration ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/foreign_key_index_requirement.png) from Pathwright).
Since we will be using foreign key columns to retrieve information, it is important to index them.
In our case since we are going to use creator_id in many places while retrieving events, we can make add it as an index as follows:
```ruby
add_index :events, :creator_id
```
 Read more on why this is important:
* https://semaphoreci.com/blog/2017/05/09/faster-rails-is-your-database-properly-indexed.html
* https://sqlperformance.com/2012/11/t-sql-queries/benefits-indexing-foreign-keys.